sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkSpanBuilder.java

```diff
@@ -152,6 +152,17 @@ public SpanBuilder setAttribute(String key, boolean value) {
     return this;
   }
 
+  @Override
+  @SuppressWarnings("unchecked")
+  public SpanBuilder setAttributes(Attributes attributes) {
```

Comment:
can you put some javadoc on this method explaining why we override the default (for performance, fewer allocations, etc)?